nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/util/TestJdbcHugeStream.java

```diff
@@ -30,14 +38,7 @@
 import java.sql.Statement;
 import java.util.Random;
 
-import org.apache.avro.file.DataFileStream;
-import org.apache.avro.generic.GenericDatumReader;
-import org.apache.avro.generic.GenericRecord;
-import org.apache.avro.io.DatumReader;
-import org.junit.BeforeClass;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.TemporaryFolder;
+import static org.junit.Assert.assertEquals;
```

Comment:
Do we still need this test class here? There's another TestJdbcHugeStream in nifi/nifi-nar-bundles/nifi-extension-utils/nifi-database-utils/src/test/java/org/apache/nifi/util/db

Reply:
I tried to remove everything, but I think some other tests in standard-processors use the method(s) here to load up a test DB. I remember needing some stuff in both places, and didn't want to try to refactor the tests vs the utils. I can take a closer look if you like

Reply:
I understand that. I've submitted a PR to your branch to refactor this. Please check.
https://github.com/mattyb149/nifi/pull/5